### PR TITLE
WebGLRenderer: Refactor creation of transmission render target.

### PR DESCRIPTION
**Pull request by Mugen87**
Related issue: https://github.com/mrdoob/three.js/pull/22731#issuecomment-1034110601

**Description**

This PR sets the size of the transmission render target to half of the drawing buffer's size. It also makes `MeshPhysicalMaterial.transmission` to a WebGL 2 only property so it's easier use mipmap generation. Besides, the transmission render target's texture now uses `LinearFilter` to its `magFilter` property.

The PR also fixes issue 5 in #22009.

### CODE REVIEW

File: docs/api/en/materials/MeshPhysicalMaterial.html [hunk 1]
```diff
@@ -188,7 +188,7 @@ <h3>[property:Float transmission]</h3>
 
 		The transmission property can be used to model these materials.<br />
 
-		When transmission is non-zero, [page:Material.opacity opacity] should be set to *1*.
+		When transmission is non-zero, [page:Material.opacity opacity] should be set to *1*. This feature can only be used with a WebGL 2 rendering context.
```
Comment (WestLangley) [inline docs/api/en/materials/MeshPhysicalMaterial.html:191]: These two sentences are unrelated. For clarity, I'd put the WebGL 2 limitation on a separate line.

Comment (Mugen87) [reply docs/api/en/materials/MeshPhysicalMaterial.html]: Makes sense! Let me update the PR.
